### PR TITLE
Cache static assets longer

### PR DESCRIPTION
**Pull request by jpmartinspt**

### CODE REVIEW

File: webapp/config.py [hunk 1]
```diff
@@ -24,3 +24,6 @@ class ConfigurationError(Exception):
     raise ConfigurationError("`WEBAPP` is not configured")
 
 WEBAPP_EXTRA_HEADERS = {}
+
+SEND_FILE_MAX_AGE_DEFAULT = 10*365*24*60*60
```
Comment (hatched) [inline webapp/config.py]: Can you add a comment as to how long this is for?

Comment (jpmartinspt) [reply webapp/config.py]: Done!

**Comment by barrymcgee** [comment]
Still to do...

- Check in all templates that static assets are being properly fingerprinting.

File: webapp/config.py [hunk 2]
```diff
@@ -24,3 +24,6 @@ class ConfigurationError(Exception):
     raise ConfigurationError("`WEBAPP` is not configured")
 
 WEBAPP_EXTRA_HEADERS = {}
+
+# Ten years default cache time on static files
+SEND_FILE_MAX_AGE_DEFAULT = 10 * 365 * 24 * 60 * 60
```
Comment (tbille) [inline webapp/config.py]: is this used somewhere?

**Pull request merged**